pypesto/store/read_from_hdf5.py

```diff
@@ -111,13 +111,24 @@ def read(self, objective: ObjectiveBase = None) -> Problem:
         problem:
             A problem instance with all attributes read in.
         """
+        # raise warning that problem is not to be used but
+        # only stores information
+        logger.warning('WARNING: You are loading a problem.\nThis problem'
+                       ' is not to be used without a separatly created'
+                       ' objective.\nIn place of problem.objective you can'
+                       ' access the Information\nstored on the objective.\n')
+
         # create empty problem
         if objective is None:
             objective = Objective()
         problem = Problem(objective, [], [])
 
         with h5py.File(self.storage_filename, 'r') as f:
             for problem_key in f['/problem']:
+                if problem_key == 'objective_info':
```

Comment:
I would rather not at all read the "objective info" into the problem.objective, but rather offer a different function reader.get_objective_info() for the moment. The problem.objective should only hold a proper objective.